src/reducers/systemIntakesReducer.ts

```diff
@@ -1,19 +1,45 @@
-import { STORE_SYSTEM_INTAKES } from '../constants/actions';
-import { SystemIntakesState } from '../types/systemIntake';
+import { SystemIntakesState } from 'types/systemIntake';
+import { prepareSystemIntakeForApp } from 'data/systemIntake';
+import { fetchSystemIntakes } from 'routines/routines';
+import { DateTime } from 'luxon';
 
 const initialState: SystemIntakesState = {
-  systemIntakes: []
+  systemIntakes: [],
+  loaded: false,
```

Comment:
By convention, this should be named `isLoaded`. I think I understand why it's in the past tense, but traditionally "load" is used in the present tense for  loading screens. While we don't have requirements for a loading screen, we can probably check `systemIntakes` for length in replacement of `isLoaded` and add `isLoading`.

https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig

Reply:
I think we need this variable to differentiate "still loading" and "it loaded and returned no intakes"

Reply:
It's not 100% apparent to me why this is necessary. After `isLoading` changes from `true` to `false`, (with no error messages) that tells us that we got all the data. If there's nothing, we load up an empty form.

Reply:
The expected place where a person has never created an intake before is an area where we expect an empty list, and that would be considered still a successful request. I can't think of any request errors where we'd want to change the `isLoading` or `isLoaded` values, atm. That's not to say there won't be for other features/areas of the UI